.vscode/settings.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
should we add the extensions.json file that recommends what plugins to also install for vscode users?

Reply:
ac82ddd